entrypoint.js

```diff
@@ -1,13 +1,13 @@
 const { Toolkit } = require('actions-toolkit')
 const { WebClient } = require('@slack/web-api')
 
-const { fetchUnreleasedCommits, linkifyPRs } = require('./utils')
+const { fetchUnreleasedCommits, linkifyPRs, getSupportedBranches } = require('./utils')
 
```

Comment:
_(nit)_ removes magic number below

```suggestion

  const NUM_SUPPORTED = 4 // 3 stable branches + 1 beta
```